### PR TITLE
Upgrading xmonad setup with taffybar

### PR DESCRIPTION
**Pull request by norrs**
File based on Ondřej Súkup @mimi1vx work

### CODE REVIEW

File: dot.xmonad/xmonad.hs [hunk 1]
```diff
@@ -30,29 +59,189 @@ myKeys conf = M.fromList
...
+  pagerHints $
+  def {terminal = "urxvt"
+                ,borderWidth = 2
+                ,modMask = mod4Mask
+                ,workspaces = myWorkspaces
+                ,layoutHook = myLayout
+                ,manageHook = myManageHook
+                ,handleEventHook = myEventHook
+                ,startupHook = myStartupHook
+                ,keys = keys defaultConfig <+> myKeys } `additionalKeys`
```
Comment (norrs) [inline dot.xmonad/xmonad.hs:230]: This is probably wrong, doesn't work :(

File: dot.xmonad/xmonad.hs [hunk 2]
```diff
@@ -30,29 +59,189 @@ myKeys conf = M.fromList
...
+------------------------------------------------------------------------
+-- myStartupHook
+myStartupHook :: X ()
+myStartupHook =
+  do setDefaultCursor xC_left_ptr
+     -- setWMName "LG3D"
+     spawnOnce "xrdb -merge ~/.Xdefaults"
+     spawnOnce "xsetroot -solid black"
+     spawnOnce "status-notifier-watcher"
+     spawnOnce "taffybar"
```
Comment (norrs) [inline dot.xmonad/xmonad.hs]: These programs doesn't boot..

**Review by norrs — COMMENTED** [review]
cc @mimi1vx :p

File: dot.xmonad/xmonad.hs (hunk 2 above)
Comment (mimi1vx) [reply dot.xmonad/xmonad.hs]: L161 and L162 can be changed to `spawn`

**Comment by norrs** [comment]
@norrs: can I ping myself to make a notification on my own?

File: dot.xmonad/xmonad.hs (hunk 1 above)
Comment (mimi1vx) [reply dot.xmonad/xmonad.hs:230]: remove keys + myKeys

**Comment by norrs** [comment]
Merging this in favor of https://github.com/norrs/dotfiles/pull/1 :P

**Pull request merged**